config/env/dev.json

```diff
@@ -26,6 +26,7 @@
           "port": "@@ETH_RPC_PORT",
           "url": "@@ETH_RPC_URL"
         },
+        "tranactionTimeoutSecs": "@@ETH_TXN_TIMEOUT",
```

Comment:
Note to self mostly, 60s with 3 retries is too long if we are anchoring every 2 minutes on dev. We'll run into a nonce issue if we timeout.